src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -61,10 +63,21 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
   sys.addShutdownHook(shutdown())
 
   lazy val db = DatabaseUtil.db
+
+  //bring slick-based repositories in scope
+  val repos = new SlickRepositories
+  import repos._
+
+  //define all datastore services
+  val tezosFees = new FeeOperations
+  val tezosStore = new TezosDatastore
```

Comment:
Why those two aren't defined as objects?

Reply:
you mean why to use a new instance or why not
```scala
object TezosFees extends FeeOperations
object TezosStore extends TezosDatastore
```?

The latter seem unfamiliar to me, and I don't know if it might work with implicit params.
The former needs to take the implicits into accounts.

Did I miss the question?